odoo/__init__.py

```diff
@@ -8,7 +8,10 @@
 # odoo must be a namespace package for odoo.addons to become one too
 # https://packaging.python.org/guides/packaging-namespace-packages/
 #----------------------------------------------------------
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
+import pkgutil
+import os.path
+__path__ = [os.path.abspath(path) for path in
+            pkgutil.extend_path(__path__, __name__)]
```

Comment:
Either put the entire listcomp on a single line, or have the entire "for in" on the same line:
```python
__path__ = [
    os.path.abspath(path)
    for path in pkgutil.extend_path(__path__, __name__)
]
```